packages/xs-vat-worker/src/console.js

```diff
@@ -1,7 +1,7 @@
 /** console for xs platform */
 const harden = x => Object.freeze(x, true);
```

Comment:
All these stubbed forms of `harden` must disappear asap. The real `harden` from SES works after `lockdown`